source/manage/cloud-data-export.rst

```diff
@@ -1,8 +1,138 @@
-Cloud Data Export
-=========================
+Mattermost Workspace Migration
+==============================
 
-If, after completing your 14-day free Cloud trial, you've decided to self-host your own Mattermost deployment, you may want to migrate your Cloud data to your self-hosted instance.
```

Comment:
I wonder if the phrase "Mattermost instance" is better here (and I see this is used elsewhere)